src/client/components/CardGroup.js

```diff
@@ -1,11 +1,20 @@
 import React, { Component } from "react";
+import PropTypes from "prop-types";
 
 function CardGroupHeader({ title }) {
   return <h3>{title}</h3>;
 }
 
+CardGroupHeader.propTypes = {
+  title: PropTypes.string
+};
+
 function CardGroup({ children }) {
   return <div>{children}</div>;
 }
 
+CardGroup.propTypes = {
+  children: PropTypes.array
```

Comment:
Please make this prop required. Additionally you can use `propTypes.shape` to check the objects inside the array. It would be nice to make sure that you are passing `DataCard`s into the `CardGroup`.

Reply:
that means that i have to make DataCard prop also required right ?